src/NuGet.Core/NuGet.Commands/project.json

```diff
@@ -50,6 +50,11 @@
       "frameworkAssemblies": {
         "System.Xml": "",
         "System.Xml.Linq": ""
+      },
+      "compilationOptions": {
+        "define": [
+          "IS_DESKTOP"
+        ]
       }
     },
     "net46": {

```

Comment:
XPlat is net46 on purpose and really just for testing, it only ships as netcoreapp1.0.

Commands looks like it has net46 for build preprocessors, but that is no longer supported in CLI and could go away.